server/base/application.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017 The Absolute Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import express from 'express';
+import requireAll from 'require-all';
+
+/**
+ * Application class
+ */
+export default class Application {
+  /**
+   * @constructor
+   */
+  constructor() {
+    if (this._instance !== undefined) {
+      throw new Error('Singleton');
+    }
+
+    this._app = express();
+    this._instance = this;
+  }
+
+  /**
+   * @param {String} path
+   */
+  start(path) {
+    requireAll({
+      dirname: path,
+      filter: /\.router\.js$/,
+      recursive: true,
+    });
+
+    this._app.listen(3000);
```

Comment:
Done.

After this patch, we block the https. The previous way is not good to me.
I'll fix it in follow-up CL.